server/ai_process.go

```diff
@@ -1056,7 +1056,10 @@ func submitLiveVideoToVideo(ctx context.Context, params aiRequestParams, sess *A
 	defer completeBalanceUpdate(sess.BroadcastSession, balUpdate)
 
 	// Send request to orchestrator
-	resp, err := client.GenLiveVideoToVideoWithResponse(ctx, req, paymentHeaders)
+	reqTimeout := 5 * time.Second
+	reqCtx, cancel := context.WithTimeout(ctx, reqTimeout)
+	defer cancel()
+	resp, err := client.GenLiveVideoToVideoWithResponse(reqCtx, req, paymentHeaders)
```

Comment:
Fun fact that I think this will timeout when the O does not have a container warmed up and ready to go. And I think this is great!

Reply:
Ahh, yes, right!